backend/cypress_test.ini

```diff
@@ -91,7 +91,7 @@ auth_type = internal
 # User auth token validity in seconds (used to interfaces like web calendars)
 user.auth_token.validity = 604800
 # user reset_password token validity (default to 900s -> 15 minutes)
-user.reset_password.validity = 900
+user.reset_password.token_validity = 900
```

Comment:
I suggest to rename to `token_lifetime` which seems to be more explicit. @inkhey @buxx @PhilippeAccorsi what do you think?

Reply:
I prefer that also.

Reply:
I prefer that also.

Reply:
`token_lifetime` +1

Reply:
I agree.

Reply:
Change done.